MainV2.cs

```diff
@@ -398,6 +398,11 @@ public static MAVLinkInterface comPort
             {
                 if (_comPort == value)
                     return;
+
+                // Disable speech on old port and enable it on the new one
+                if (_comPort != null) _comPort.speechenabled = false;
+                if (value != null) value.speechenabled = true;

```

Comment:
what if it was disabled before hand?

Reply:
Currently, we don't disable this on any interfaces that we also set as `MainV2.comPort`. I can't imagine a case where we want some interfaces to speak but don't want the currently selected interface to speak (if you want no interfaces to speak, this doesn't override the global `Speech.speechEnable`)

I can definitely see that the hard-coding of false on the old port, is pretty limiting though.